package.json

```diff
@@ -88,7 +88,7 @@
         "@dagrejs/dagre": "^1.1.4",
         "@eslint/compat": "^1.1.0",
         "@fluentui-contrib/react-data-grid-react-window": "^1.2.0",
-        "@fluentui/react-components": "^9.64.0",
+        "@fluentui/react-components": "^9.72.2",
```

Comment:
Any reason for the update. Not saying we shouldn't just curious what this brings in.